pkg/polaris/azure.go

```diff
@@ -344,7 +344,7 @@ func (c *Client) AzureSubscriptionSetName(ctx context.Context, id AzureSubscript
 		Name: name,
 	}}
 
-	status, err := c.gql.AzureCloudAccountUpdate(ctx, graphql.CloudNativeProtection, nil, nil, sub)
+	status, err := c.gql.AzureUpdateCloudAccount(ctx, graphql.CloudNativeProtection, nil, nil, sub)
 	if len(status) != 1 {
```

Comment:
Should `err != nil` be checked ahead of the status length?

Reply:
Agree, should be fixed now.